src/app/components/About/About.jsx

```diff
@@ -18,12 +19,17 @@ class Sidebar extends React.Component {
     super(props);
 
     this.state = { location: {} };
+    this.gaEvent = this.gaEvent.bind(this);
   }
 
   componentDidMount() {
     this.setState({ location: window.location });
   }
 
+  gaEvent(type) {
```

Comment:
I would re-name this to be gaSocialMediaEvent since it's very specific to social media link clicks.. GOAL!